downstream/modules/automation-hub/con-remote-repos.adoc

```diff
@@ -4,5 +4,5 @@
 
 By configuring remote repositories, you can set your {PrivateHubName} to synchronize Red Hat Certified Collections hosted in your organization's repository on `{Console}` and your choice of collections in {Galaxy}.
 
-Each remote repository located in menu:Repo Management[Remote] provides information for both the *community* and *rh-certified* repository about when the repository was *last updated* and when content was *last synced*. 
-You can add new content to {HubNameMain} at any time using the *Edit* and *Sync* features included on the menu:Repo Management[Remote] page.
\ No newline at end of file
+Each remote repository located in menu:Automation Hub[Remotes] provides information for both the *community* and *rh-certified* repository about when the repository was *last updated* and when content was *last synced*.
```

Comment:
remote repository -> remote configuration. These aren't technically repositories, they're closer to "credentials" in controller. It's a configuration that repositories can use to pull content from a remote.

I don't believe "last synced" is shown on this page anymore.

Reply:
The text has been updated as suggested.